staging/src/k8s.io/component-base/metrics/wrappers.go

```diff
@@ -65,6 +68,70 @@ type GaugeMetric interface {
 	SetToCurrentTime()
 }
 
+// GaugeMetricVec is a collection of Gauges that differ only in label values.
+// This is really just one Metric.
+// It might be better called GaugeVecMetric, but that pattern of name is already
+// taken by the other pattern --- which is treacherous.  The treachery is that
```

Comment:
Taken by what?
I see "CounterVecMetric" which seems better than what you have here.

Reply:
friendly ping